packages/manager/modules/pci/src/projects/new/payment/components/credit/index.html

```diff
@@ -0,0 +1,62 @@
+<div class="pci-project-new-payment-credit">
+    <p data-translate="pci_project_new_payment_credit_explain"></p>
+
+    <div class="d-flex justify-content-between mb-3">
+        <oui-radio
+            class="oui-radio_btn"
+            data-ng-repeat="amount in $ctrl.predifinedAmounts track by $index"
+            data-name="credit"
+            data-thumbnail
+            data-variant="light"
+            data-model="$ctrl.model.credit"
+            data-value="amount"
+            data-on-change="$ctrl.onPredifinedAmountRadioChange()"
+            data-disabled="$ctrl.globalLoading.finalize"
+        >
+            <span data-ng-bind="amount.text"></span>
+        </oui-radio>
+        <oui-radio
+            class="oui-radio_btn"
+            data-name="credit"
+            data-thumbnail
+            data-variant="light"
+            data-model="$ctrl.otherAmount.radio"
+            data-value="true"
+            data-on-change="$ctrl.onOtherAmountRadioChange()"
+            data-disabled="$ctrl.globalLoading.finalize"
+        >
+            <span
+                data-translate="pci_project_new_payment_credit_amount_other"
+            ></span>
+        </oui-radio>
+    </div>
+
+    <div data-ng-if="$ctrl.otherAmount.radio">
+        <div
```

Comment:
Don't the `oui-field` component suit the needs ?

Reply:
oui-field doesn't work well with getterSetter `ngModelOptions` :-$